Makefile

```diff
@@ -54,7 +54,12 @@ endif
 USE_NINJA ?= 1
 ifeq ($(USE_NINJA), 1)
 ifneq ($(shell which ninja), )
-GENERATOR=-GNinja -DMAX_LINK_JOBS=$(MAX_LINK_JOBS) -DMAX_HIGH_MEM_JOBS=$(MAX_HIGH_MEM_JOBS)
+GENERATOR := -GNinja
+GENERATOR += -DMAX_LINK_JOBS=$(MAX_LINK_JOBS)
+GENERATOR += -DMAX_HIGH_MEM_JOBS=$(MAX_HIGH_MEM_JOBS)
+
+# Ninja makes compilers disable colored output by default.
+GENERATOR += -DVELOX_FORCE_COLORED_OUTPUT=ON
```

Comment:
This works for now but we should add a CMakePresets.json at some point to better integrate with IDEls, at that point we probably want to move this into the cml.

Reply:
I'm not sure I get how this would go, but I'll defer this to a future refactor :)

Reply:
By checking [`CMAKE_GENERATOR`](https://cmake.org/cmake/help/latest/variable/CMAKE_GENERATOR.html#cmake-generator) :)